cmd/deploy.go

```diff
@@ -20,6 +20,8 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const airflowURLType = "airflow"
```

Comment:
I feel like this constant should be part of the `houston` package since it strictly relates to houston deployment URLs.

Reply:
moved to houston package